content/posts/planning-checklist.md

```diff
@@ -0,0 +1,184 @@
+---
+title: planning checklist for people in tech
+description: this post tries to add a method to madness which runs amok every 3 months and both leaders and teams run helter-skelter to put an exciting list of okrs together to executing them
+layout: layouts/post.njk
+date: 2024-05-13
+tags:
+  - leadership
+  - planning
+---
+
+it's almost may and in just about a month a brand new quarter would start which comes with a possibility of endless opportunities in engg roadmap parlance. 
+
+we've come a long way from knowing what would get done years in advance, like a construction building, say a new mall for which once the blueprint is finalised, you can usually estimate the multi-year work upfront at least on paper before you dive deep into execution. but who knew building software is even tougher or have we deliberately made it that way?
+
+we work in sprints, we put our heads down for two weeks and then again and again. some teams decide within these two weeks what next to do and sort of groom their stories with no hints or indication of what lies ahead and coins words like agile :P
+
+but there's just too much flux in this way. there's so much extra time being spent deciding every two weeks what to do. does it even matter right now or can be pushed out?
+
+in real world, work happens around big projects and launches. and they have dates or an expected date of launch. everything begins from there and you usually back calculate.
+
+so with above context, how do you decide what gets done in a quarter? what actually is **planning**?
+
+> you can wing it yolo style because life's too short to plan anything, because seriously who cares?
+
+**actually you should, as if your life depends on it**
+
+i've done it both ways (just don't tell my boss though), planned (roadmaps, sprints) on the fly and derived them meticulously by using the below framework. the difference was monumental in the quality of deliverables and the morale of the team!
+
+earlier i'd have a meeting every two weeks, rethinking priorities and not just me, my entire team. i thought having them on the call would give them visibility but all they saw was uncertainty and jitteriness. they felt as if nobody had any idea what exactly needed to be done, and everyone was busy pitching things they would like to do instead which might not be in the best interest of the organisation. like everyone can't work on reducing build times or trying out a new framework which came out yesterday!
```

Comment:
optional: the nobody and everyone confused me, maybe more words for some clarity 

```suggestion
earlier i'd have a meeting every two weeks, rethinking priorities and not just me, my entire team. i thought having them on the call would give them visibility but all they saw was uncertainty and jitteriness. they felt as if nobody had any idea what exactly needed to be done, and the entire team was busy pitching things they would like to do instead which might not be in the best interest of the organisation. doesn’t make sense for everyone to work on reducing build times or trying out a new framework which came out yesterday!
```